Use `flutter pub add` for dependencies in networking cookbooks (#8729)
flutter/website: src/cookbook/networking/delete-data.md

## 1. Add the `http` package

To install the `http` package,

> add it to the dependencies section of the `pubspec.yaml` file
> by running `flutter pub add`:

Reviewer: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```
